framework/src/mysql.js

```diff
@@ -232,6 +233,11 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 			query.whereIn(property, values);
 		}
 
+		if (params.whereJsonSupersetOf) {
+			const { property, values } = params.whereJsonSupersetOf;
+			query.whereJsonSupersetOf(property, values);
```

Comment:
Does this work when the array (in `property` column) only contains a subset of the entries in the `values` array?